Dockerfile.multi-stage

```diff
@@ -1,6 +1,6 @@
 # By default we pin to amd64 sha. Use make docker to automatically adjust for arm64 versions.
 ARG BASE_DOCKER_SHA="14d68ca3d69fceaa6224250c83d81d935c053fb13594c811038c461194599973"
-FROM golang:1.16-alpine3.12 as builder
+FROM golang:1.17-alpine3.15 as builder
```

Comment:
This change also seems unrelated. Probably just needs a rebase since this exact version is used in `main`